templates/plate-chain/plate_chain/prompts.py

```diff
@@ -0,0 +1,70 @@
+AI_1536W_RESPONSE = '[{"row_start": 2, "row_end": 33, "col_start": 1, "col_end": 48, "contents": "SB_ID"}]'  # noqa: E501
+
+USER_EXAMPLE_DICT = {
+    24: HUMAN_24W_PROMPT,
+    96: HUMAN_96W_PROMPT,
+    384: HUMAN_384W_PROMPT,
+    1536: HUMAN_1536W_PROMPT,
+}
+
+AI_REPONSE_DICT = {
+    24: AI_24W_RESPONSE,
+    96: AI_96W_RESPONSE,
+    384: AI_384W_RESPONSE,
+    1536: AI_1536W_RESPONSE,
+}
+
+
+def create_prompt(num_plates: int | None, num_rows: int | None, num_cols: int | None):
```

Comment:
from future annotaitons isn't imported -- so the pipe syntax will break with older python versions, missing annotations on return types